### PR TITLE
[WIP] Global accessibility awareness day

### PR DESCRIPTION
**Pull request by toddjordan**
Install ember-a11y-testing
Install ember-a11y for focusing-outlet
clean up axe errors
decorate filter behavior with aria labels to help with screen readers

This is the start to an accessibility section in the tutorial.

excuse the messy diff, dependent on 2 prs in front of it. (acceptance test, style)

### CODE REVIEW

File: app/templates/components/list-filter.hbs [hunk 1]
```diff
@@ -1,2 +1,4 @@
-City: {{input value=value key-up=(action 'handleFilterEntry')}}
+<label for="filterByCity" class="screen-reader">Filter By City</label>
+  {{input id="filterByCity" value=value key-up=(action 'handleFilterEntry') class="light" placeholder="Filter By City" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="rental-list"}}
```
Comment (stefanpenner) [inline app/templates/components/list-filter.hbs]: Is it possible for us to instead just use an input tag? Much simpler implementation, and is literally what the DOM already provides no abstraction.

Comment (toddjordan) [inline app/templates/components/list-filter.hbs]: yeah, as long was we can provide an action for key-up.

Comment (stefanpenner) [inline app/templates/components/list-filter.hbs]: https://developer.mozilla.org/en-US/docs/Web/Events/keyup is part of the DOM, so it would just work

Comment (toddjordan) [inline app/templates/components/list-filter.hbs]: so implement a keyUp event handler in the component.  Can do.  Sorry, its late :-P

Comment (stefanpenner) [inline app/templates/components/list-filter.hbs]: no just `<input onkeyup={{action ...` no component needed

Comment (toddjordan) [inline app/templates/components/list-filter.hbs]: I thought that too and tried with initial lack of success.  will take another look. thx

Comment (toddjordan) [inline app/templates/components/list-filter.hbs]: without the input helper, what is the best way to provide the value to the action?

Comment (stefanpenner) [inline app/templates/components/list-filter.hbs]: http://miguelcamba.com/blog/2016/01/24/ember-closure-actions-in-depth/ should provide the required info

Comment (toddjordan) [inline app/templates/components/list-filter.hbs]: thx, got it `onkeyup={{action "handleFilterEntry" value="target.value"}}`

**Comment by jenweber** [comment]
Closing this due to significant overhaul in the following PRs/issues:
https://github.com/ember-learn/super-rentals/pull/87
https://github.com/ember-learn/ember-cli-tutorial-style/pull/2
https://github.com/ember-learn/super-rentals/issues/86
https://github.com/emberjs/guides/issues/2152

Based on Slack discussion, we're paring this down to the bare minimum changes - enough to introduce a11y to users and give them pointers to the right places to keep learning.

**Pull request closed without merging**